python/pyspark/mllib/clustering.py

```diff
@@ -867,18 +940,20 @@ class StreamingKMeansModel(KMeansModel):
     1
     """
 
-    def __init__(self, clusterCenters, clusterWeights):
+    def __init__(self, clusterCenters: List["VectorLike"], clusterWeights: "VectorLike"):
         super(StreamingKMeansModel, self).__init__(centers=clusterCenters)
-        self._clusterWeights = list(clusterWeights)
+        self._clusterWeights = list(clusterWeights)  # type: ignore[arg-type]
```

Comment:
Ideally, this and similar ignores wouldn't be needed, but we have `List[VectorLike]`, `np.ndarray` and `List[np.ndarray]` in different places (including test outputs) and I am concerned that making things consistent can break some 3rd party code.